config/scheduler/settings.ini

```diff
@@ -273,6 +273,9 @@ pa.scheduler.core.defaultgroupfilename=config/authentication/group.cfg
 #	- "SchedulerPAMLoginMethod" to use PAM login management
 pa.scheduler.core.authentication.loginMethod=SchedulerFileLoginMethod
 
+# Creates a credential file (username.cred) for each successful login in the authentication folder
+pa.scheduler.create.credentials.when.login=true
```

Comment:
if the property default value is set to false, why setting it in the default config file to true ?